doc/user/content/integrations/http-api.md

```diff
@@ -170,6 +170,15 @@ type SqlRequest = Simple | Extended;
 interface Notice {
 	message: string;
 	severity: string;
+	detail?: string;
+	hint?: string;
+}
+
+interface Error {
+	message: string;
+	code: string;
```

Comment:
This is going to be really useful, can you point me to where these are defined?

Reply:
https://www.postgresql.org/docs/current/errcodes-appendix.html

We roughly attempt to support these. Note the code in this PR hard codes all of these to a single state, so you can't use them yet. (We have code that does this, but it's in the wrong place and we have to move it.)

Reply:
ah ok, thanks!